lib/tasks/default-tasks/index.js

```diff
@@ -0,0 +1,82 @@
+import Listr from 'listr';
```

Comment:
Why is this folder called `default-tasks` and not `release-tasks`?

Reply:
Good point, I just thought, as `sgr` is the default command, I name the tasks running in the default command `default-tasks`.

Maybe I rename the default command to `release command` and also rename the tasks.